### PR TITLE
Fixes #35100 - disable submit if no changes in roles edit

### PR DESCRIPTION
**Pull request by Ron-Lavi**
also adjusting the edit roles modal's width to 50% instead of 70%

### CODE REVIEW

File: webpack/components/AnsibleHostDetail/components/RolesTab/EditRolesModal/EditRolesForm.js [hunk 1]
```diff
@@ -50,7 +51,7 @@ const EditRolesForm = props => {
       key="confirm"
       variant="primary"
       onClick={() => callMutation({ variables })}
-      isDisabled={loading}
+      isDisabled={loading || isEqual(initAvailableOptions, availableOptions)}
```
Comment (stejskalleos) [inline webpack/components/AnsibleHostDetail/components/RolesTab/EditRolesModal/EditRolesForm.js:54]: For me button is still enabled, steps to reproduce:
* (host without any assigned roles)
* Show modal => button is disabled :heavy_check_mark: 
* Select role(s) => button is enabled :heavy_check_mark: 
* Remove all assigned roles => button is enabled  :x:

Comment (stejskalleos) [reply webpack/components/AnsibleHostDetail/components/RolesTab/EditRolesModal/EditRolesForm.js:54]: Yop, here it is:

https://user-images.githubusercontent.com/59385976/176603099-8252d8c8-a26a-47da-a024-8d035dbea97a.mp4

**Review by stejskalleos — CHANGES_REQUESTED** [review]
Found another issue:
* Assign three roles to the host and save
* Go back to Edit roles modal
* Change order of assigned roles >> Button is still disabled

**Comment by Ron-Lavi** [comment]
Thanks for double-checking @stejskalleos !
can you try again?

**Review by stejskalleos — APPROVED** [review]
:green_apple: **LGTM**, works as expected.
Thanks @Ron-Lavi !

**Pull request merged**